### ISSUE TITLE
Background events don't fire eventClick in timeGrid

### ISSUE THREAD

**Issue by bfwg** [fullcalendar/fullcalendar#5579]
change one of the events in [codepen](https://codepen.io/arshaw/pen/VweMByK?editable=true&editors=001) to 
```
      {
        start: '2020-05-02T15:00',
        end: '2020-05-02T15:00',
        overlap: false,
        display: 'background'
      },
```
Follow by changing the calendar to the week view. Clicking on the background time slot won't trigger the click event.
Am I missing something or this is a valid problem.

**Comment by acerix** [comment]
Thanks, I can see it's not firing in timeGrid view, is that the issue you mean?

https://codepen.io/acerix/pen/RwrxgGN?editors=001

**Comment by bfwg** [comment]
Yes

**Comment by kamilchlebek** [comment]
@acerix I've got the same issue with `eventMouseEnter`.

Looks like both are caused by incorrect `z-index` for background events. 

Setting `z-index:2` for `.fc .fc-timegrid-col-bg` fixes the issue. (I'm not sure about further consequences after changing z-index)

**Comment by arshaw** [comment]
Issues like this will be fixed in v5.2.0: https://codepen.io/arshaw/pen/OJMKvZw?editors=001

**Comment by iraeborn** [comment]
> Issues like this will be fixed in v5.2.0: https://codepen.io/arshaw/pen/OJMKvZw?editors=001

I use v5.11 and the events still dont't fire in eventClick on react.

**Comment by floofydoug** [comment]
I am also not able to get an eventClick to fire from timeGridDay with resources.  Tried going back to v5.2.0 but unfortunately that version does not support React 17.
